Bottom: 030d1e2873e6a83256b30cc08db81505fba45366
Top:    b5c2a779072ac43912983f2422c7c3e5bea7f396
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-27 16:31:09 -0700

Refresh of rstream-conn

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 4ecd2ce..428b273 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -317,11 +317,11 @@ static int sync_test(void)
 {
 	int ret;
 
-	ret = dst_addr ? send_xfer(rs, 4) : recv_xfer(rs, 4);
+	ret = dst_addr ? send_xfer(4) : recv_xfer(4);
 	if (ret)
 		return ret;
 
-	return dst_addr ? recv_xfer(rs, 4) : send_xfer(rs, 4);
+	return dst_addr ? recv_xfer(4) : send_xfer(4);
 }
 
 static int run_test(void)
